Makefile.include

```diff
@@ -337,11 +337,17 @@ endif
 endif
 endif # BUILD_IN_DOCKER
 
+# Check given command is available in the path
```

Comment:
Is it possible to move it in a separate `.mk` file in `makefiles` directory so it can be easily imported by other makefiles ? I think it could also be usefull for #8573.

Reply:
Folo!